relations.csv

```diff
@@ -4698,3 +4698,11 @@ HGNC,SENP8,isa,FPLX,SENP
 FPLX,DESI,isa,FPLX,Desumoylase
 HGNC,DESI1,isa,FPLX,DESI
 HGNC,DESI2,isa,FPLX,DESI
+HGNC,CCNH,partof,FPLX,CDK_activating_kinase_complex
+HGNC,CDK7,partof,FPLX,CDK_activating_kinase_complex
+HGNC,MNAT1,partof,FPLX,CDK_activating_kinase_complex
+HGNC,TIMM10,partof,FPLX,TIM22_complex

```

Comment:
Please review https://www.ebi.ac.uk/complexportal/complex/CPX-6124 for potential missing members of this complex and a nice equivalence